cmd/db-manager/v1beta1/main.go

```diff
@@ -73,12 +73,12 @@ func (s *server) Check(ctx context.Context, in *health_pb.HealthCheckRequest) (*
 func main() {
 	flag.Parse()
 	var err error
-	dbNameEnvName := common.DBNameEnvName
-	dbName := os.Getenv(dbNameEnvName)
-	if dbName == "" {
-		klog.Fatal("DB_NAME env is not set. Exiting")
+	dbPasswordEnvName := common.DBPasswordEnvName
+	dbPassword := os.Getenv(dbPasswordEnvName)
```

Comment:
Maybe we can omit `dbPasswordEnvName` var ?
```suggestion
	dbPassword := os.Getenv(common.DBPasswordEnvName)
```